src/ota_flash.c

```diff
@@ -74,7 +74,7 @@ bool mg_ota_end(void) {
   bool ok = false;
   if (s_size) {
     size_t size = s_addr - base;
-    uint32_t crc32 = mg_crc32(0, base, s_size);
+    uint32_t crc32 = mg_crc32(0, base + MG_FLASH_OFFSET, s_size);
```

Comment:
Is this macro really needed ? (see other files, I don't in which order GH will present this to you...)

Reply:
Solved, see above